src/main/java/de/dentrassi/crypto/pem/PemUtils.java

```diff
@@ -112,6 +113,10 @@ private static void loadFrom(final Map<String, Entry> result, final String alias
 
                 key = converter.getKeyPair((PEMKeyPair) object).getPrivate();
 
+            }else if(object instanceof PrivateKeyInfo) {
```

Comment:
I think there is some whitespace issue again, and it would be great if you could remove the commented out, empty line.